launch/client.py

```diff
@@ -577,6 +577,21 @@ def create_model_endpoint(
                 - Otherwise, if the average number of concurrent requests per worker is higher
                   than ``per_worker``, then the number of workers will be increased to meet the elevated traffic.
 
+                Note that there is a duality to the number of workers (which we'll represent using ``max_workers`` here)
+                and the value of ``per_worker``. This can be illustrated by considering two extreme scenarios - in both
+                scenarios, let ``T`` be the target throughput for completing the entire request workload, and let ``c``
```

Comment:
Hm is there a reason why they'd be different in these two contexts? IIRC it was a conscious decision (to me at least) to unify the meaning of `per_worker` for both sync and async endpoints, which aligns w/ the implementation anyway.

Reply:
I suppose they're largely the same? `per_worker` is pretty much the constant of proportionality between the number of workers and some queue length somewhere (either the celery queue or some queue in the sync workers that might have a small `max_length`)

although things have the potential to get weird with the interaction of `per_worker` with these queues that can functionally have small `max_length` (because of the concurrency limit thing) but that's a whole other thing